torch/testing/_comparison.py

```diff
@@ -102,7 +105,8 @@ def default_tolerances(
             raise TypeError(
                 f"Expected a torch.Tensor or a torch.dtype, but got {type(input)} instead."
             )
-    rtols, atols = zip(*[_DTYPE_PRECISIONS.get(dtype, (0.0, 0.0)) for dtype in dtypes])
+    dtype_precisions = dtype_precisions or _DTYPE_PRECISIONS
```

Comment:
Instead of hardcoding `_DTYPE_PRECISIONS` here, this allows us to also use this function for `assert_allclose`, since that uses different default precisions.